src/WebUI/Common/KubeCardWithTable.tsx

```diff
@@ -227,5 +227,5 @@ export function renderPodsStatusTableCell(
         </Tooltip>
     ) : null;
 
-    return renderTableCell(rowIndex, columnIndex, tableColumn, itemToRender, undefined, "bolt-table-cell-content-with-link");
+    return renderTableCell(rowIndex, columnIndex, tableColumn, itemToRender, undefined, podsCountString ? "bolt-table-cell-content-with-link" : "");
```

Comment:
https://github.com/Microsoft/azpipelines-kubernetesUI/blob/7255f8b2cd312435198a360289bddbd649f5d105/src/WebUI/Common/KubeCardWithTable.tsx#L172

in this link, we do send the status of the POD, we have a discussion to consider for liveness probe also. we will be adding it little later.

Reply:
will raise another PR for this